Report/Chapters/Process.tex

```diff
@@ -28,17 +28,17 @@ \section{Code tools}
 
 \subsection{Maven}
 
-Larger Java projects usually have dependencies. These dependencies can also have dependencies. To make sure every team member was using the same versions of the libraries the project was no doubt going to use, we decided to use Maven, a Java build tool. Using Maven as a standardized build process also allowed team members to work from their preferred environment: the back end crew preferred IntelliJ IDEA by Jetbrains, while the front end developers worked mostly in Eclipse. \index{Maven}
+Larger Java projects usually have dependencies. These dependencies can also have dependencies. To make sure every team member was using the same versions of the libraries the project was no doubt going to use, we decided to use Maven, a Java build tool. Using Maven as a standardized build process also allowed team members to work from their preferred environment: the back end crew preferred IntelliJ IDEA by JetBrains, while the front end developers worked mostly in Eclipse. \index{Maven}
```

Comment:
These dependencies can *in turn* also have dependencies.

> To make sure every team member was using the same versions of the libraries the project was no doubt going to use, we decided to use Maven, a Java build tool

Hard to read sentence